universe-instances-extended/universe-instances-extended.cabal

```diff
@@ -38,6 +38,19 @@ library
   exposed-modules:  Data.Universe.Instances.Extended
   build-depends:
       adjunctions    >=4.3 && <4.5
-    , base           >=4.3 && <4.13
     , comonad        >=4.2 && <5.1
+    , containers
     , universe-base  >=1.1 && <1.1.2
+
+  if impl(ghc >=8.6.1)
+    build-depends:   base           >=4.12 && <4.13
```

Comment:
This is not right.

See https://oleg.fi/gists/posts/2019-06-03-compat-packages.html#s:3